src/fundus/publishers/ch/__init__.py

```diff
@@ -17,3 +19,13 @@ class CH(metaclass=PublisherGroup):
             Sitemap("https://www.srf.ch/new-news-sitemap"),
         ],
     )
+
+    NZZ = Publisher(
+        name="Neue Zürcher Zeitung (NZZ)",
+        domain="https://www.nzz.ch/",
+        parser=NZZParser,
+        sources=[
+            NewsMap("https://www.nzz.ch/sitemap/news0.xml"),
+            Sitemap("https://www.nzz.ch/sitemap.xml", sitemap_filter=inverse(regex_filter(r"sitemap/[\d]{4}/[\d]{2}"))),
```

Comment:
There would also be some available RSS Feeds: https://www.nzz.ch/information/rss-ld.105364

Reply:
I would argue here that those are not necessary here, judging by the news map.